src/Model/Profile.php

```diff
@@ -45,6 +45,25 @@ public static function getByUID($uid)
 		return $profile;
 	}
 
+	/**
+	 * @brief Returns the profile based on a ID
+	 *
+	 * @param int $uid The User ID
+	 * @param int $id The id of the profile (optional)
+	 * @param array $fields The fields to retrieve
+	 *
+	 * @return array Array of profile data
+	 * @throws \Exception
+	 */
+	public static function select(int $uid, int $id = null, array $fields = [])
+	{
+		if (empty($id)) {
+			return DBA::selectToArray('profile', $fields, ['uid' => $uid]);
```

Comment:
I'd rather you did away with the single `Profile::select` method and instead have `Profile::getListByUser` and `Profile::getById`.

Reply:
Sounds reasonable ;-)